src/dynamixel_main.cpp

```diff
@@ -0,0 +1,123 @@
+#include <fcntl.h>
+#include <termios.h>
+#define STDIN_FILENO 0
+
+#include <stdio.h>
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "dynamixel_sdk/dynamixel_sdk.h"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#define PROTOCOL_VERSION                2.0
+
+#define DXL_ID                          1
+#define BAUDRATE                        57600
+#define DEVICENAME                      "/dev/ttyUSB0"
+
+#define NODE_NAME                       "dynamixel_ping"
+#define TOPIC_NAME                      "dynamixel_topic"
+
+using namespace std;
+
+int getch()
+{
+  struct termios oldt, newt;
+  int ch;
+  tcgetattr(STDIN_FILENO, &oldt);
+  newt = oldt;
+  newt.c_lflag &= ~(ICANON | ECHO);
+  tcsetattr(STDIN_FILENO, TCSANOW, &newt);
+  ch = getchar();
+  tcsetattr(STDIN_FILENO, TCSANOW, &oldt);
+  return ch;
+}
+
```

Comment:
Better put it inside the `main()` function, instead of a separate function. If you intended to separate it, put it inside a class on another file.